<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="noInvoiceLabel">
      <property name="text">
       <string>No Invoice</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noInvoiceLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tanggalLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tanggalLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="kodeAdvertiserLabel">
      <property name="text">
       <string>Kode Advertiser</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="kodeAdvertiserComboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="kodeIklanLabel">
      <property name="text">
       <string>Kode Iklan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="kodeIklanComboBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="frekwensiLabel">
      <property name="text">
       <string>Frekwensi</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="frekwensiLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="jumlahBayarLabel">
      <property name="text">
       <string>Jumlah Bayar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="jumlahBayarLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="pPHLabel">
      <property name="text">
       <string>PPH</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="pPHLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="totalBayarLabel">
      <property name="text">
       <string>Total Bayar</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="totalBayarLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="terbilangLabel">
      <property name="text">
       <string>Terbilang</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="terbilangLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Biaya Bayar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ubahButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simpanButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>601</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>No Invoice</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kode Advertiser</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kode Iklan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Frekwensi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Bayar</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PPH</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total bayar</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Terbilang</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cetakButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
